app/cafe.py

```diff
@@ -0,0 +1,31 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import NotWearingMaskError
+from app.errors import OutdatedVaccineError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if "vaccine" not in visitor:
+            try:
+                raise NotVaccinatedError
+            except NotVaccinatedError:

```

Comment:
there is no need in the try-except block. Raise an error with an informative message